std_msgs/Header header
int16[16] raw_channels
int16[16] mapped_channels
bool failsafe
bool frame_lost
